string image_filename
float32 approxRange
bool firstTarget
---
int32 centerX
int32 centerY
int32 totalX
float32 rangeSquared
bool newImageReceived

